lib/turkish_cities/elevation.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class Elevation
+  include DecomposerHelper
+
+  file_path = File.join(File.dirname(__FILE__), 'data/cities.yaml')
+  CITY_LIST = YAML.load_file(file_path)
+
+  def find_by_elevation(type, elevation_one, elevation_two)
+    return [] unless check_number_range(elevation_one, 4, 1923)
```

Comment:
Did you forgot to update the min-max values here? It doesn't make sense to limit it between 4-1923. I feel like we don't even need to limit the range at all, is there a specific reason? 🤔

Reply:
It's actually coming from data. City with lowest elevation data is Kocaeli with 4 meters and highest is Erzurum with 1923. Making a search call outside this spec won't make sense. Maybe more robust solution can be an exception with message to user

Reply:
Then how does the examples for this method on Readme really output some cities?
e.g: https://github.com/sarslanoglu/turkish_cities/pull/115/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R400

Reply:
Oh my bad. Readme is outdated. I will update asap

Reply:
And an exception just like you explained above in your message would be nice 💯